media/css/firefox/whatsnew/whatsnew-rocket.scss

```diff
@@ -6,89 +6,120 @@
-    padding: 0;
+    overflow: hidden;
+    position: relative;
+    text-align: center;
 }
 
-.rocket-content {
-    padding-bottom: 40px;
+.rocket-intro {
+    @include font-size-level3;
+    position: relative;
+    z-index: 2;
+}
 
-    @media #{$mq-desktop} {
-        background: url('/media/img/firefox/whatsnew/rocket/bg-rocket-left.png') left -60px bottom no-repeat,
-                    url('/media/img/firefox/whatsnew/rocket/bg-rocket-right.png') right bottom no-repeat;
-        background-size: 880px 540px, 210px 315px;
-    }
+.rocket-text {
+    margin: 0 auto;
+    max-width: 540px;
+}
+
+.rocket-code {
+    background-color: #fff;
+    border-radius: 20px;
+    padding: 25px;
+    display: inline-block;
+    margin: 0 auto;
 
-    @media #{$mq-desktop-wide} {
-        background-position: left -30px bottom, right bottom;
+    img {
+        display: block;
     }
+}
 
-    @media only screen and (-webkit-min-device-pixel-ratio: 1.5) and (min-width: 1000px),
-           only screen and (min-resolution: 1.5dppx) and (min-width: 1000px),
-           only screen and (min-resolution: 144dpi) and (min-width: 1000px) {
+@media #{$mq-desktop} {
 
-        background-image: url('/media/img/firefox/whatsnew/rocket/bg-rocket-left-high-res.png'),
-                          url('/media/img/firefox/whatsnew/rocket/bg-rocket-right-high-res.png');
+    .rocket-body {
+        background: linear-gradient(to bottom, #73F6D8, #7950DC);
     }
-}
 
-.content-intro {
-    margin: 0 auto;
-    max-width: $width-tablet;
-    padding: 30px 20px 20px;
-    text-align: center;
+    .rocket-content {
+        &:before,
+        &:after {
+            border-radius: 100%;
+            content: '';
+            display: block;
+            height: 888px;
+            position: absolute;
+            width: 888px;
+            z-index: 1;
+        }
+
+        &:before {
+            background: linear-gradient(to bottom, #6efcea 40%, #7c6fe4);
+            right: calc(50% - 80px);
+            top: -302px;
+        }
+
+        &:after {
+            background: linear-gradient(to bottom, #70cde5, #7292e7 40%);
+            bottom: -552px;
+            left: calc(50% - 220px);
+        }
+    }
 
-    p {
-        margin-top: 10px;
+    .rocket-body {
+        text-align: left;
+
+        .content {
+            padding: 40px;
+        }
     }
-}
 
-.moz-logo {
-    position: absolute;
-    right: 20px;
-    top: 0;
-}
+    .rocket-logo {
+        margin-bottom: 10px;
+    }
+
+    .rocket-intro {
+        @include font-size(32px);
+        background: url('/media/img/firefox/whatsnew/rocket/screenshot.png') left center no-repeat;
+        background-size: 370px 553px;
+        padding: 20px 0 20px 420px;
+
+        @media #{$mq-high-res} {
+            background-image: url('/media/img/firefox/whatsnew/rocket/screenshot-high-res.png');
+        }
+    }
 
-.content-intro {
-    @include font-size-level4;
+    .rocket-text {
+        margin: 0 0 10px 0;
```

Comment:
10px seems like a really tight bottom margin for this text. Even though it's exactly the same as the logo's bottom margin, on screen there's a lot more space above the text than below it and they should be about even. Maybe bump this bottom margin up to 30px, or 1em if you want to go proportional. And it has 0 bottom margin on small screens so it's extra cramped. I'd probably make it 1em for all viewports. 

Could also use a bit of space between the QR code and the Play button when they stack on small screens.

![image](https://user-images.githubusercontent.com/205591/39654518-fcfee766-4fa9-11e8-82d3-18031775828f.png)